tools/ci/check_grep.sh

```diff
@@ -303,6 +303,12 @@ do
     done < <(jq -r '[.map_file] | flatten | .[]' $json)
 done
 
+# Check for non-515 compatable .proc/ syntax
+if grep -P --exclude='__byond_version_compat.dm' '\.proc/' code/**/*.dm; then
+    echo "ERROR: Outdated proc reference use detected in code, please use proc reference helpers."
```

Comment:
```suggestion
    echo "${RED}ERROR: Outdated proc reference use detected in code, please use proc reference helpers.${NC}"
```

Consistency with the rest of check_grep.sh (colors provide good contrast)